<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConversionParameterDialog</class>
 <widget class="QDialog" name="ConversionParameterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>423</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Conversion Parameters</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images.qrc">
    <normaloff>:/app/icons/qwinff_small.png</normaloff>:/app/icons/qwinff_small.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_audio">
        <attribute name="title">
         <string>Audio</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QCheckBox" name="chkDisableAudio">
           <property name="text">
            <string>Disable Audio</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupAudioOptions">
           <property name="title">
            <string>Audio Options</string>
           </property>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="lblSampleRate">
                <property name="text">
                 <string>Sample Rate</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cbAudioSampleRate">
                <property name="editable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lblHz">
                <property name="text">
                 <string>Hz</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="2" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Channels</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinChannels">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>2</number>
                </property>
                <property name="value">
                 <number>2</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="3" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_12">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Volume</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinVolume">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>200</number>
                </property>
                <property name="value">
                 <number>100</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>%</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="lblAudioBitrate">
                <property name="text">
                 <string>Bitrate</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinAudioBitrate">
                <property name="buttonSymbols">
                 <enum>QAbstractSpinBox::NoButtons</enum>
                </property>
                <property name="maximum">
                 <number>9999</number>
                </property>
                <property name="value">
                 <number>64</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lblAudioBitrateUnit">
                <property name="text">
                 <string>kb/s</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="1">
             <widget class="QCheckBox" name="chkAutoAdjustBitrate">
              <property name="toolTip">
               <string>Use the source bitrate if it is smaller than the configured output bitrate. This can reduce output file size in certain conditions.</string>
              </property>
              <property name="text">
               <string>Auto Adjust Bitrate</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_video">
        <attribute name="title">
         <string>Video</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QCheckBox" name="chkDisableVideo">
           <property name="text">
            <string>Disable Video</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupVideoOptions">
           <property name="title">
            <string>Video Options</string>
           </property>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>Width</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinWidth">
                  <property name="buttonSymbols">
                   <enum>QAbstractSpinBox::UpDownArrows</enum>
                  </property>
                  <property name="suffix">
                   <string> px</string>
                  </property>
                  <property name="maximum">
                   <number>9999</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Height</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinHeight">
                  <property name="buttonSymbols">
                   <enum>QAbstractSpinBox::UpDownArrows</enum>
                  </property>
                  <property name="suffix">
                   <string> px</string>
                  </property>
                  <property name="maximum">
                   <number>9999</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item row="2" column="0">
             <widget class="QCheckBox" name="chkVideoSameQuality">
              <property name="toolTip">
               <string>Use same video quality as source (implies VBR).</string>
              </property>
              <property name="text">
               <string>Same Quality as Source</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="1">
               <widget class="QSpinBox" name="spinCropTop">
                <property name="buttonSymbols">
                 <enum>QAbstractSpinBox::UpDownArrows</enum>
                </property>
                <property name="suffix">
                 <string> px</string>
                </property>
                <property name="maximum">
                 <number>9999</number>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QSpinBox" name="spinCropLeft">
                <property name="buttonSymbols">
                 <enum>QAbstractSpinBox::UpDownArrows</enum>
                </property>
                <property name="suffix">
                 <string> px</string>
                </property>
                <property name="maximum">
                 <number>9999</number>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="lblCrop">
                <property name="text">
                 <string>Crop</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QSpinBox" name="spinCropRight">
                <property name="buttonSymbols">
                 <enum>QAbstractSpinBox::UpDownArrows</enum>
                </property>
                <property name="suffix">
                 <string> px</string>
                </property>
                <property name="maximum">
                 <number>9999</number>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QSpinBox" name="spinCropBottom">
                <property name="buttonSymbols">
                 <enum>QAbstractSpinBox::UpDownArrows</enum>
                </property>
                <property name="suffix">
                 <string> px</string>
                </property>
                <property name="maximum">
                 <number>9999</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Bitrate</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinVideoBitrate">
                <property name="buttonSymbols">
                 <enum>QAbstractSpinBox::NoButtons</enum>
                </property>
                <property name="maximum">
                 <number>9999</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>kb/s</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="3" column="0">
             <widget class="QCheckBox" name="chkDeinterlace">
              <property name="text">
               <string>Deinterlace</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Time</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QGroupBox" name="groupCutting">
           <property name="title">
            <string extracomment="video time cutting: options for begin time and end time">Cutting</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QCheckBox" name="chkFromBegin">
              <property name="text">
               <string>From Begin</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="timeBegin">
              <property name="currentSection">
               <enum>QDateTimeEdit::HourSection</enum>
              </property>
              <property name="displayFormat">
               <string notr="true">hh:mm:ss</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chkToEnd">
              <property name="text">
               <string>To End</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="timeEnd">
              <property name="currentSection">
               <enum>QDateTimeEdit::HourSection</enum>
              </property>
              <property name="displayFormat">
               <string notr="true">hh:mm:ss</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupScaling">
           <property name="title">
            <string extracomment="time scaling, changing the speed of the output file">Scaling</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="lblSpeed">
                <property name="text">
                 <string>Speed (Experimental)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="spinSpeedFactor">
                <property name="suffix">
                 <string notr="true">%</string>
                </property>
                <property name="minimum">
                 <double>10.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>1000.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_advanced">
        <attribute name="title">
         <string>Advanced</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>FFmpeg</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QLabel" name="lblFFmpegOptions">
                <property name="text">
                 <string>Additional FFmpeg Options</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPlainTextEdit" name="txtFFmpegOptions"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>chkDisableAudio</tabstop>
  <tabstop>cbAudioSampleRate</tabstop>
  <tabstop>spinAudioBitrate</tabstop>
  <tabstop>chkAutoAdjustBitrate</tabstop>
  <tabstop>spinChannels</tabstop>
  <tabstop>spinVolume</tabstop>
  <tabstop>spinWidth</tabstop>
  <tabstop>spinHeight</tabstop>
  <tabstop>spinVideoBitrate</tabstop>
  <tabstop>chkVideoSameQuality</tabstop>
  <tabstop>chkDeinterlace</tabstop>
  <tabstop>spinCropTop</tabstop>
  <tabstop>spinCropLeft</tabstop>
  <tabstop>spinCropRight</tabstop>
  <tabstop>spinCropBottom</tabstop>
  <tabstop>txtFFmpegOptions</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>chkDisableVideo</tabstop>
 </tabstops>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConversionParameterDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConversionParameterDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkDisableAudio</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupAudioOptions</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkDisableVideo</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupVideoOptions</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
